components/compass-runtime-agent/README.md

```diff
@@ -32,6 +32,7 @@ The Runtime Agent uses these environment variables:
 - **APP_CLUSTER_CERTIFICATES_SECRET** specifies the Namespace and the Name of the Secret in which to store the client certificate and the key.
 - **APP_CA_CERTIFICATES_SECRET** specifies the Namespace and the Name of the Secret in which to store the CA certificate.
 - **APP_SKIP_COMPASS_TLS_VERIFY** specifies whether to communicate with the Connector Service and the Director with disabled TLS verification.
+- **APP_SKIP_APPS_TLS_VERIFY** specifies whether applications synchronized from Compass will be setup to communicate with external systems with disabled TLS verification.
```

Comment:
```suggestion
- **APP_SKIP_APPS_TLS_VERIFY** specifies whether applications synchronized from Compass will be set up to communicate with external systems with disabled TLS verification.
```
- `setup` is a noun, the verb is `set up`. 
- By `application` we DON'T mean the representation of the external system here, right?